src/Web/Internal/HttpApiData.hs

```diff
@@ -640,7 +647,7 @@ instance FromHttpApiData ZonedTime where parseUrlPiece = runAtto Atto.zonedTime
 -- Right 2015-10-03 00:14:24 UTC
 instance FromHttpApiData UTCTime   where parseUrlPiece = runAtto Atto.utcTime
 
-instance FromHttpApiData NominalDiffTime where parseUrlPiece = fmap fromInteger . parseUrlPiece
+instance FromHttpApiData NominalDiffTime where parseUrlPiece = fmap (realToFrac :: F.Pico -> NominalDiffTime) . parseUrlPiece
```

Comment:
I believe `realToFrac` is not as efficient as `secondsToNominalDiffTime`.
Can you do the same trick as with `nominalDiffTimeToSeconds`?